benches/linkbench/app.lua

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env tarantool
```

Comment:
Should not this script be a part of the benchmark itself? I don't know, but curious.

Reply:
Now the whole benchmark scripts are a big mess, that needs fixing, however I would like to do that in the future. [See](https://github.com/tarantool/bench-run/pull/14#discussion_r546418427)